### PR TITLE
new branch and pr for expose kafka

### PR DESCRIPTION
**Pull request by ranum**
New branch and PR for just the expose-kafka service.

### CODE REVIEW

File: services/expose-kafka/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# expose-kafka
+
+A deployable service to assist with kafka interaction.
+
+## operations
+
+- listing topics-this will provide a list of all topics
+current registered with the kafka broker
+
+    (GET)  https://\<kafkahelperbaseurl:port>
```
Comment (ibmatc) [inline services/expose-kafka/README.md:10]: Needs to be updated from kafkahelper to expose-kafka

**Review by ibmatc — APPROVED** [review]
One minor comment, but LGTM

File: services/expose-kafka/README.md (hunk 1 above)
Comment (ranum) [reply services/expose-kafka/README.md:10]: Ack, how did I forget that.

Comment (ranum) [reply services/expose-kafka/README.md]: fixed

**Pull request merged**